cpr/session.cpp

```diff
@@ -792,8 +803,13 @@ void Session::Impl::prepareCommon() {
 
 #if LIBCURL_VERSION_MAJOR >= 7
 #if LIBCURL_VERSION_MINOR >= 21
-    /* enable all supported built-in compressions */
-    curl_easy_setopt(curl_->handle, CURLOPT_ACCEPT_ENCODING, "");
+    if (acceptEncoding_.has()) {
```

Comment:
Shouldn't this be:
```suggestion
    if (!acceptEncoding_.has()) {
```

Reply:
Sorry for that XD
Now I changed it to empty()